it/_naming.md

```diff
@@ -27,9 +23,7 @@ Come per molti linguaggi, suggerisco di nominare le variabili costanti completam
 
 {% include snippets/naming/02/index.html %}
 
-###### Approfondimenti
-
-* [Dealing With Constants in Sass](http://www.sitepoint.com/dealing-constants-sass/)
+If you really want to play with the ideas of constants in Sass, you should read [this dedicated article](http://www.sitepoint.com/dealing-constants-sass/).
```

Comment:
This sentence was missing. I added it in English, but it would need to be translated. :)